### DISCUSSION TITLE
плагины

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#25] [Q&A]
в стим фикс входит помимо неоф. обновления еще и набор плагинов.

**Comment by Kor-Angar** [comment]
Плагин который заменяет у ряда(или всех) предметов  "name" на  "description"

![image](https://user-images.githubusercontent.com/49514764/167412960-9a3296b5-5ee7-48fc-81a6-5dddc51626ee.png)

**Reply by Kor-Angar** [reply]
@dosinabox  подскажи пожалуйста как его отключи в неоф. обновление установленном через мастерскую стима. в g2a_nr_scriptpatch_ru.ini?

**Reply by dosinabox** [reply]
Плагин называется zMiscUtils.
Подробные описания отключаются в Gothic.ini:
```
NameToDescCats=|
; ... for items from listed categories the value of 'description' field will be used as focus name
; example: 'Scroll' -> 'Firebolt'
; all possible categories: NF|FF|MUN|ARMOR|FOOD|DOCS|POTION|LIGHT|RUNE|MAGIC
; leave single symbol '|' to disable the feature
```

**Comment by dosinabox** [comment]
Надо будет вынести всё в отдельный том
